### ISSUE TITLE
Add Support to Provider Wasabi (would be included as S3 Compatible)

### ISSUE THREAD

**Issue by wanderleihuttel** [Kyrodan/KeeAnywhere#130]
Hello
Would be interesting include new providers "S3 compatibles".
One provider that is possible to free use for 30 days and you can storage 1TB of data is www.wasabi.com.

Docs are in: https://wasabi.com/help/docs/

Best Regards
Wanderlei

**Comment by Kyrodan** [comment]
Please vote for this issue (via reaction or comment), if you are interested to get it integrated in KeeAnywhere

**Comment by Kyrodan** [comment]
Seems easy to integrate: Wasabi is 100% compatible to AWS-SDK. So just have to add an option for own url.

**Comment by Tupsi** [comment]
> Seems easy to integrate: Wasabi is 100% compatible to AWS-SDK. So just have to add an option for own url.

Thats why I stumbled over this plugin this morning. Thought "just look for a keepass plugin which can access AWS and your good to go". Turns out that just a custom url option is missing as every AWS Region is currently hard coded into the plugin.
Would be very nice to add. But seeing now that this is already an issue from almost three years ago, I wont get my hopes up :-)
